### ISSUE TITLE
How are Accounts related to the wallet

### ISSUE THREAD

**Issue by lucode** [OCA/vertical-community#22]
Hi,
I don't understand how the COM accounts are related to the wallet.
What is puzzling me is that this account are in CHF and not COM. Aren't any COM accounts possible in the odoo Accounts.
![2014-12-06 14_51_44-accounts - odoo](https://cloud.githubusercontent.com/assets/2928740/5327607/d6a1d8fc-7d58-11e4-87fe-ae854b4e4b9c.png)

**Comment by YannickB** [comment]
Hello Lucas,

Each currency need to use the account required by the legislations. For COM there is no legislations so I just created some dummy account outside of the official chart account.

I suggest you check what I have done in the module l10n_fr_wallet, this is were I configure the wallet for using € for transaction in France. I suggest you do the same kind of module to manage transactions in CHF. Do not hesitate to ask if you need more detailed informations.

**Comment by lucode** [comment]
But are this accounts now in CHF or in COM?

**Comment by florent-valeureux** [comment]
These accounts are in COM. But since we are using the main accounting module. Community currencies are indicated within the system currency

If the company currency is set in Euro it will appear in Euro or in whatever currency.
![gypoxm2 1](https://cloud.githubusercontent.com/assets/9918969/5421234/8de9a01a-8260-11e4-9b85-491b28fa0130.png)
Here I switch defaut currency
![gzbaevw 1](https://cloud.githubusercontent.com/assets/9918969/5421264/f7eff360-8260-11e4-8b3f-3b6384f2e8cc.png)

But I joined you that it will be more elegant to spell them in COM. But will be legal without enabling multiple currencies features ? I dont know.

**Comment by YannickB** [comment]
Yes the currency in account.account doesn't really matter, you can change them as you wish.
